renderer/markdown.go

```diff
@@ -145,6 +145,10 @@ func (m *MarkdownRenderer) RenderFieldDoc(text string) string {
 	// so that including | in a comment does not result in wonky tables.
 	out := strings.ReplaceAll(text, "|", "\\|")
 
+	// Escape the curly bracket character.
+	out = strings.ReplaceAll(out, "{", "\\{")
+	out = strings.ReplaceAll(out, "}", "\\}")
```

Comment:
We do the same thing in `RenderDefault` but I think it's okay to duplicate. We will see in the future if we share some of the chars to be escaped between the two functions.